### DISCUSSION TITLE
How do I run pre-releasese?

### DISCUSSION THREAD

**Discussion by mxgrn** [KosmosisDire/obsidian-webpage-export#335] [Q&A] [answered]
Any instructions on how to use a pre-release?

**Comment by KosmosisDire** [comment, marked as answer]
Hey, not sure if you figured it out since this is closed, but the easiest way is to install the BRAT plugin and add a new beta plugin using `https://github.com/KosmosisDire/obsidian-webpage-export` this url.

**Reply by mxgrn** [reply]
Yeah, sorry, I was too slow to figure out that I could download the zip manually 🤦‍♂️
